frontend/packages/ux-editor/src/components/Properties/EditSubformTableColumns/ColumnElement/EditColumnElement/EditColumnElement.tsx

```diff
@@ -68,9 +90,13 @@ export const EditColumnElement = ({
   };
 
   const selectComponent = (values: string[]) => {
-    const selectedComponentId = values[0];
-    const selectedComponent = availableComponents.find((comp) => comp.id === selectedComponentId);
+    const componentId = values[0];
+    setSelectedComponentId(componentId);
+
+    const selectedComponent = availableComponents.find((comp) => comp.id === componentId);
+    if (!selectedComponent) return;
 
+    selectComponentBinding(selectedComponent);
     const binding = convertDataBindingToInternalFormat(selectedComponent, 'simpleBinding');
```

Comment:
We should probably replace the `simpleBinding` string here with the value from the new dropdown list. This will update/save the value in the `cellContent` property

```suggestion
    const binding = convertDataBindingToInternalFormat(selectedComponent, dataModelBinding);
```

Reply:
Fixed 👍